pymeasure/instruments/keithley/keithley2602B.py

```diff
@@ -80,44 +109,41 @@ def get_trigger_event_description_strings():
 
 
 class Keithley2600DigitalIOPin:
-    def __init__(self, instrument, pin_number):
+    def __init__(self, instrument: Keithley2602B, pin_number: int) -> None:
         self.instrument = instrument
         self.pin_number = pin_number
 
-    def ask(self, cmd):
+    def ask(self, cmd: str) -> Union[str, None]:
```

Comment:
So I get that, if the ask fails, we _could_ get `None` as a response, and so maybe `Union[str, None]` is as general as possible, but I also wonder if it's more correct to just say the return type is `str`, as anything else indicates that something went wrong.

Reply:
I agree, I will change this